README.md

```diff
@@ -0,0 +1,127 @@
+# Openshift APIs for Data Protection (OADP) Python Client Wrapper 
+
+Manage Openshift adp Custom Resources 
+
+Read more about OADP
+
+https://access.redhat.com/articles/5456281
+
+
+## Installation
+
+
+```bash
+# Install from repo
+git clone https://github.com/oadp-qe/openshift-adp-python-wrapper
+pip install <path to>/openshift-adp-python-wrapper
+```
+
+## Usage Example
+### Creating a dpa CR, wait for a successful reconciliation and delete it. 
+note: 
+1. The j2 helper module includes a build-in filter for getting env variable values: `env_override`
```

Comment:
I think it's better to read settings from a file and not as env vars, as it will only apply to the very shell process, so you would have export it for any shell process (either from terminal or IDE)

Reply:
@mperetzred @amastbau Can we add a shell script or function here it will take user input and generate a sample settings file for us. We can provide basic information like (BSL, VSL cloud provider) and depending on user input we will generate a sample settings file. What do you think ?